lib/entities/content-type-fields.ts

```diff
@@ -15,11 +16,32 @@ interface RegExp {
   flags: string
 }
 
+interface NodesValidation {
+  [BLOCKS.EMBEDDED_ENTRY]?: Pick<
+    ContentTypeFieldValidation,
+    'size' | 'linkContentType' | 'message'
+  >[]
+  [INLINES.EMBEDDED_ENTRY]?: Pick<
+    ContentTypeFieldValidation,
+    'size' | 'linkContentType' | 'message'
+  >[]
+  [INLINES.ENTRY_HYPERLINK]?: Pick<
+    ContentTypeFieldValidation,
+    'size' | 'linkContentType' | 'message'
+  >[]
+  [BLOCKS.EMBEDDED_ASSET]?: Pick<ContentTypeFieldValidation, 'size' | 'message'>[]
+  [INLINES.ASSET_HYPERLINK]?: Pick<ContentTypeFieldValidation, 'size' | 'message'>[]
+  [BLOCKS.EMBEDDED_RESOURCE]?: {
+    validations: Pick<ContentTypeFieldValidation, 'size' | 'message'>[]
+    allowedResources: ContentTypeAllowedResources
```

Comment:
Me again with the same question: This should also be an array, right?

Reply:
Thank you for reviewing so thoroughly, you are right again!